README.md

```diff
@@ -19,10 +21,53 @@ UserSimCRS follows the architecture of a typical task-based dialogue system, whi
 
 We refer to the [documentation](https://iai-group.github.io/UserSimCRS/) for details. Specifically, see [this page](https://iai-group.github.io/UserSimCRS/setup_agent.html) on how to set up an existing agent to be evaluated using UserSimCRS.
 
+## Installation
+
+The recommended version of Python is 3.9.  
+The easiest way to install UserSimCRS and all of its dependencies is by using pip:
+```shell
+python -m pip install -r requirements/requirements.txt
+```
+To work on the documentation you also need to install other dependencies:
+```shell
+python -m pip install -r requirements/docs_requirements.txt
+```
+## Usage 
+
+A YAML configuration file is necessary to start the MovieBot, see [default configuration](config/default/config_default.yaml) as example.  
```

Comment:
Nit: 
- ", see" -> "; see"
- "as example" => "for an example"

Reply:
Done.